src/app/components/Onboard/Welcome/features.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+export default function Features(props) {
```

Comment:
Destructure props so other devs know what props to give it. In this case it would be really helpful to add proptypes, add a comment, or somehow document which properties the elements of the feature array should/could have.

Reply:
wish we used typescript. but i'm not going to push it at this point 😅